### PR TITLE
Current shuttle is no longer stated on the "Status" panel

### PR DESCRIPTION
**Pull request by Core0verload**
Why? Because **SURPRISE MOTHERFUCKERS!**

If meme shuttles are buyable, this is an issue, and making "Status" panel show current shuttle name is **not a fix for it**. If meme shuttles are not buyable and appear only when badmins want to have fun, it's not an issue at all. 

If you want something that can't be changed by players, use pods.

### CODE REVIEW

**Comment by Haevacht** [comment]
Fuck off, meteor kills enough as is.

**Comment by GunHog** [comment]
That was a feature to help warn people about meme shuttles, like the asteroid. It would be wise to keep it.

**Comment by Jordie0608** [comment]
For reference, the PR in which this was added #21704

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -525,7 +525,7 @@ var/next_mob_id = 0
 			stat(null, "Next Map: [nextmap.friendlyname]")
 		stat(null, "Server Time: [time2text(world.realtime, "YYYY-MM-DD hh:mm")]")
 		if(SSshuttle.emergency)
-			stat(null, "Current Shuttle: [SSshuttle.emergency.name]")
+			//stat(null, "Current Shuttle: [SSshuttle.emergency.name]")
```
Comment (Jordie0608) [inline code/modules/mob/mob.dm]: Don't comment code out, just remove

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

**Comment by Core0verload** [comment]
If meme shuttles are buyable, this is another issue, and making "Status" panel show current shuttle name is not a fix for it. If meme shuttles are not buyable and appear only when badmins want to have fun, it's not an issue at all. 

If you want something that can't be changed by players, use pods.

**Comment by MrPerson** [comment]
Stat panels are terrible. The more shit that's gone from them, the better

**Comment by improvedname** [comment]
Last time i was playing people were standing in escapr i looked in status panel and screamed ASTEROID SHUTTLE GET OUT OF ESCAPR then they got crushed its not like its helping anyways altough putting it on those screens that show ETA be neat

**Comment by Grazyn** [comment]
If you want to remove meme shuttles why not just make a PR that makes them unbuyable? I'm not really a fan of this new practice of making a feature worse and worse until it's finally removed instead of pushing a removal PR in the first place.

**Comment by KorPhaeron** [comment]
I don't really have a particular love of the shuttle being in the status panel, but this was a compromise for the people who were sad about being hit by asteroids a feature requested by admins (maybe head admins? I don't remember exactly) and reigniting old drama probably isn't worth it.

Also feature freeze

**Comment by improvedname** [comment]
Just add this to shuttle E.T.A screens later when you examine them like showing the eta current shuttle and bridge messages

**Comment by kevinz000** [comment]
Feature freeze

**Comment by Grazyn** [comment]
>Just add this to shuttle E.T.A screens later when you examine them like showing the eta current shuttle and bridge messages

This. Less spoonfeeding, less immersion-breaking than tabs, and still gives you a way to know which shuttle is coming. There is no other way to know if you join after the announcement.

**Comment by RandomMarine** [comment]
Wouldn't hurt to log shuttle purchases in newscasters too.

**Comment by Bawhoppen** [comment]
why do you memers want dangerous shuttles purchasable but then don't want the risk associated with them of not knowing what you have?

**Comment by Grazyn** [comment]
If the intention was to have the meme shuttles come as a surprise to everyone, why is the purchase announcement public? It just seems like an oversight or something that wasn't really thought into. Giving the crew some way to know which shuttle has been bought, whether throught newscaster logs, ETA screen or anything else seems reasonable.

**Comment by MrStonedOne** [comment]
Feature.

**Review by Jordie0608 — APPROVED** [review]

**Comment by MrStonedOne** [comment]
"Purchased shuttles are now secret" sounds like a feature to me

**Comment by MrStonedOne** [comment]
>Stat panels are terrible. The more shit that's gone from them, the better
How about you back that up with an argument

**Comment by KorPhaeron** [comment]
This won't be merged unless headmins on this server change their minds, because they will disable the feature if it isn't like this.

For the record I'd prefer they not be in the status panel either

**Comment by Core0verload** [comment]
How about moving it to ETA panel examine?

**Comment by KorPhaeron** [comment]
I'd be in support of that but you'll have to ask @lzimann and the other head admins

**Comment by lzimann** [comment]
That would be nicer and more "immersive" than having it on status panel in my opinion. I'll ask the other headmins about it.

**Comment by Core0verload** [comment]
Any results?

**Comment by lzimann** [comment]
Yea we agree on being in the station displays rather than status panel.

**Pull request closed without merging**